hack/lib/test.sh

```diff
@@ -23,7 +23,12 @@ readonly   red=$(tput setaf 1)
 readonly green=$(tput setaf 2)
 
 kube::test::clear_all() {
-  kubectl delete "${kube_flags[@]}" rc,pods --all --grace-period=0 --force
+  if kube::test::if_supports_resource "rc" ; then
```

Comment:
Following the convention of other kube::test functions. Ex: kube::test::if_has_string